sanitize

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
```

Comment:
We don't use apache and logs will be rotated so this can go away as far as I can tell